### ISSUE TITLE
 TypeError str_replace(): Argument #3 ($subject) must be of type array|string, bool givenexi

### ISSUE THREAD

**Issue by kenjis** [kint-php/kint#398]
Kint 5.0

```
VENDORPATH/kint-php/kint/src/Kint.php at line 634

627         }
628 
629         return '/'.\implode('/', $file);
630     }
631 
632     public static function getIdeLink(string $file, int $line): string
633     {
634         return \str_replace(['%f', '%l'], [$file, $line], static::$file_link_format);
635     }
636 
637     /**
638      * Returns specific function call info from a stack trace frame, or null if no match could be found.
639      *
640      * @param array $frame The stack trace frame in question
641      * @param array $args  The arguments
```

https://github.com/kint-php/kint/blob/35d9d0e24c63db775a95a713099b001198c25391/init.php#L50
`\ini_get('xdebug.file_link_format')` returns `false` when Xdebug is not enabled.

**Comment by jnvsor** [comment]
Fixed in 5.0.1 - going to get CI to test both with and without xdebug to stop that happening in future. Can you confirm so I can close it?

**Comment by kenjis** [comment]
Thank you! I've confirmed.
